index.js

```diff
@@ -276,18 +279,29 @@ class K8sVMExecutor extends Executor {
 
         const podConfig = yaml.safeLoad(podTemplate);
 
+        const nodeSelectors = {};
+
         if (this.diskLabel) {
             const diskConfig = annotations[DISK_RESOURCE] || '';
-            const nodeSelectors = diskConfig.toUpperCase() === 'HIGH' ?
+            const diskSelectors = diskConfig.toUpperCase() === 'HIGH' ?
                 { [this.diskLabel]: 'high' } : {};
 
-            hoek.merge(nodeSelectors, this.nodeSelectors);
+            hoek.merge(nodeSelectors, diskSelectors);
+        }
 
-            setNodeSelector(podConfig, nodeSelectors);
-        } else {
-            setNodeSelector(podConfig, this.nodeSelectors);
+        if (this.diskSpeedLabel) {
+            const speedValues = ['turbo', 'high'];
```

Comment:
move defined constants out of the function.

Reply:
addressed